### DISCUSSION TITLE
bring back featured versions on mod homepages

### DISCUSSION THREAD

**Discussion by nellePoint** [modrinth/.github#313] [Feature requests]
this was by far the quickest way to download a mod and download the version the mod developer suggested.

the current download button feels extremely uncomfortable to use seeing as it has the suggestion to download modrinth launcher before selecting a game version, and doesnt even show featured versions. this incredibly frustrates me so i always go to the versions tab now and sift through to find the right version of the mod.

i dont use the modrinth launcher, this is one of the reasons i left curseforge.

**Comment by falseresync** [comment]
A lot of authors didn't understand featured versions well, and some used them completely wrong, trying to make "editions" of mods and other unintentional stuff. 
As for "it suggests Modrinth app" - why does that even bother you, it doesn't hide any selection buttons like CF does, it's all literally in the same modal?
But I do agree that it works pretty awfully, it doesn't even open half the time, and works worse on Firefox :p

**Reply by nellePoint** [reply]
i dont have a problem with the modrinth app being a thing or letting users know it exists, i do have a problem with it being the top option as opposed to just downloading the mod which is what im on the site to do, especially since most people use alternative launchers.

as a whole, yeah the site *does* function oddly, often buttons will just not click.

as far as the featured versions thing, i still think that its useful or something akin to it would be useful, as it was my primary way of downloading mods, and when there wasnt a featured version, it showed the latest version(s) of the mod with a download button, it was much more effecient.

nowadays i mostly download mods through the prismlauncher UI anyway, as most of the time i already know what mods im going to download and dont need to look into what they are doing very much.

**Reply by falseresync** [reply]
Out of these, I understand what you mean in all, except I still just plain don't get the problem with the Modrinth app button - since it doesn't obstruct the flow, it is equal to the other buttons UI/UX-wise and not "into your face" so to speak, and it's abundantly clear that it's *an option*, and *not* a necessity :P

**Reply by nellePoint** [reply]
ive not claimed that it seems as though a necessity.

however i do think its in your face, being the top option rather than the bottom option, and i also *mostly* have a problem with the version selector for the download to be very very slow and clunky. seeing as the loader and such is hidden until you select a game version.

and i still believe the featured versions should be right there.
